pkg/models/studio.go

```diff
@@ -56,19 +57,17 @@ type StudioReader interface {
 	Query(ctx context.Context, studioFilter *StudioFilterType, findFilter *FindFilterType) ([]*Studio, int, error)
 	GetImage(ctx context.Context, studioID int) ([]byte, error)
 	HasImage(ctx context.Context, studioID int) (bool, error)
+	AliasLoader
 	StashIDLoader
-	GetAliases(ctx context.Context, studioID int) ([]string, error)
 }
 
 type StudioWriter interface {
-	Create(ctx context.Context, newStudio Studio) (*Studio, error)
-	Update(ctx context.Context, updatedStudio StudioPartial) (*Studio, error)
-	UpdateFull(ctx context.Context, updatedStudio Studio) (*Studio, error)
+	Create(ctx context.Context, input StudioDBInput) (*int, error)
+	UpdatePartial(ctx context.Context, input StudioDBInput) (*Studio, error)
+	Update(ctx context.Context, updatedStudio *Studio) error
```

Comment:
Updated methods to match Performer, except Create returns the created id instead of accepting the studio as a pointer and updating in place. I've been hurt too many times by that pattern of not realizing my object is getting updated inside a method without me knowing, so I try to avoid it.

Reply:
Reverted my changed for Create.
Should I revert UpdatePartial as well? What's the use case for passing in the id separate instead of as part of the partial model being updated?